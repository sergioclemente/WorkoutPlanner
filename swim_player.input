(5min, 40)
(30sec, 40, easy)
(5min, 60)
(30sec, 40, easy)
(5min, 65)
(30sec, 40, easy)
10[(2min, 75), (20sec, 40, easy)]
(5min, 40, easy)
15[(30sec, 100), (20sec, 40, easy)]
(5min, 40, easy)
-- EOT
(5min, 40)
(30sec, 40, easy)
8[(30s, descend 1-4), (10s, 40)]
(30sec, 40, easy)
10[(90s, 75), (30sec, 100), (40sec, 40, easy)]
(5min, 40, easy)
10[(15s, max), (45s, easy)]
(5min, 40, easy)
-- EOT
(60s, 100, 10s, work)
-- EOT
(10s, 100, 110, 5s)
-- EOT
