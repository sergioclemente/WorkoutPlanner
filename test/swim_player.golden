Input: 
(5min, 40)
(30sec, 40, easy)
(5min, 60)
(30sec, 40, easy)
(5min, 65)
(30sec, 40, easy)
10[(2min, 75), (20sec, 40, easy)]
(5min, 40, easy)
15[(30sec, 100), (20sec, 40, easy)]
(5min, 40, easy)
Titles: 
(0;300) 40w
(300;330) easy 40w
(330;630) 60w
(630;660) easy 40w
(660;960) 65w
(960;990) easy 40w
(990;1110) 75w 1/10
(1110;1130) easy 40w 1/10
(1130;1250) 75w 2/10
(1250;1270) easy 40w 2/10
(1270;1390) 75w 3/10
(1390;1410) easy 40w 3/10
(1410;1530) 75w 4/10
(1530;1550) easy 40w 4/10
(1550;1670) 75w 5/10
(1670;1690) easy 40w 5/10
(1690;1810) 75w 6/10
(1810;1830) easy 40w 6/10
(1830;1950) 75w 7/10
(1950;1970) easy 40w 7/10
(1970;2090) 75w 8/10
(2090;2110) easy 40w 8/10
(2110;2230) 75w 9/10
(2230;2250) easy 40w 9/10
(2250;2370) 75w 10/10
(2370;2390) easy 40w 10/10
(2390;2690) easy 40w
(2690;2720) 100w 1/15
(2720;2740) easy 40w 1/15
(2740;2770) 100w 2/15
(2770;2790) easy 40w 2/15
(2790;2820) 100w 3/15
(2820;2840) easy 40w 3/15
(2840;2870) 100w 4/15
(2870;2890) easy 40w 4/15
(2890;2920) 100w 5/15
(2920;2940) easy 40w 5/15
(2940;2970) 100w 6/15
(2970;2990) easy 40w 6/15
(2990;3020) 100w 7/15
(3020;3040) easy 40w 7/15
(3040;3070) 100w 8/15
(3070;3090) easy 40w 8/15
(3090;3120) 100w 9/15
(3120;3140) easy 40w 9/15
(3140;3170) 100w 10/15
(3170;3190) easy 40w 10/15
(3190;3220) 100w 11/15
(3220;3240) easy 40w 11/15
(3240;3270) 100w 12/15
(3270;3290) easy 40w 12/15
(3290;3320) 100w 13/15
(3320;3340) easy 40w 13/15
(3340;3370) 100w 14/15
(3370;3390) easy 40w 14/15
(3390;3420) 100w 15/15
(3420;3440) easy 40w 15/15
(3440;3740) easy 40w
----------------------------
Input: 
(5min, 40)
(30sec, 40, easy)
8[(30s, descend 1-4), (10s, 40)]
(30sec, 40, easy)
10[(90s, 75), (30sec, 100), (40sec, 40, easy)]
(5min, 40, easy)
10[(15s, max), (45s, easy)]
(5min, 40, easy)
Titles: 
(0;300) 40w
(300;330) easy 40w
(330;360) descend 1-4 easy 1/8
(360;370) 40w 1/8
(370;400) descend 1-4 easy 2/8
(400;410) 40w 2/8
(410;440) descend 1-4 easy 3/8
(440;450) 40w 3/8
(450;480) descend 1-4 easy 4/8
(480;490) 40w 4/8
(490;520) descend 1-4 easy 5/8
(520;530) 40w 5/8
(530;560) descend 1-4 easy 6/8
(560;570) 40w 6/8
(570;600) descend 1-4 easy 7/8
(600;610) 40w 7/8
(610;640) descend 1-4 easy 8/8
(640;650) 40w 8/8
(650;680) easy 40w
(680;770) 75w 1/10
(770;800) 100w 1/10
(800;840) easy 40w 1/10
(840;930) 75w 2/10
(930;960) 100w 2/10
(960;1000) easy 40w 2/10
(1000;1090) 75w 3/10
(1090;1120) 100w 3/10
(1120;1160) easy 40w 3/10
(1160;1250) 75w 4/10
(1250;1280) 100w 4/10
(1280;1320) easy 40w 4/10
(1320;1410) 75w 5/10
(1410;1440) 100w 5/10
(1440;1480) easy 40w 5/10
(1480;1570) 75w 6/10
(1570;1600) 100w 6/10
(1600;1640) easy 40w 6/10
(1640;1730) 75w 7/10
(1730;1760) 100w 7/10
(1760;1800) easy 40w 7/10
(1800;1890) 75w 8/10
(1890;1920) 100w 8/10
(1920;1960) easy 40w 8/10
(1960;2050) 75w 9/10
(2050;2080) 100w 9/10
(2080;2120) easy 40w 9/10
(2120;2210) 75w 10/10
(2210;2240) 100w 10/10
(2240;2280) easy 40w 10/10
(2280;2580) easy 40w
(2580;2595) max easy 1/10
(2595;2640) easy easy 1/10
(2640;2655) max easy 2/10
(2655;2700) easy easy 2/10
(2700;2715) max easy 3/10
(2715;2760) easy easy 3/10
(2760;2775) max easy 4/10
(2775;2820) easy easy 4/10
(2820;2835) max easy 5/10
(2835;2880) easy easy 5/10
(2880;2895) max easy 6/10
(2895;2940) easy easy 6/10
(2940;2955) max easy 7/10
(2955;3000) easy easy 7/10
(3000;3015) max easy 8/10
(3015;3060) easy easy 8/10
(3060;3075) max easy 9/10
(3075;3120) easy easy 9/10
(3120;3135) max easy 10/10
(3135;3180) easy easy 10/10
(3180;3480) easy 40w
----------------------------
Input: 
(60s, 100, 10s, work)
Titles: 
(0;60) work 100w
(60;70) rest
----------------------------
Input: 
(10s, 100, 110, 5s)
Titles: 
(0;10) 100w - 110w
(10;15) rest
----------------------------
Input: 
2[(150s, alternate arm, 30s)]
Titles: 
(0;150) alternate arm easy 1/2
(150;180) rest 1/2
(180;330) alternate arm easy 2/2
(330;360) rest 2/2
----------------------------
